dso-l1/src/main/java/com/tc/object/ClientEntityManagerImpl.java

```diff
@@ -546,25 +551,30 @@ private void internalRelease(EntityID eid, EntityDescriptor entityDescriptor, Ru
 
   private InFlightMessage queueInFlightMessage(EntityID eid, Supplier<NetworkVoltronEntityMessage> message, Set<VoltronEntityMessage.Acks> requestedAcks, InFlightMonitor monitor, long timeout, TimeUnit units, boolean shouldBlockGetOnRetire) throws TimeoutException {
     InFlightMessage inFlight = new InFlightMessage(eid, message, requestedAcks, monitor, shouldBlockGetOnRetire);
-    msgCount.increment();
-    inflights.add(ClientConfigurationContext.MAX_PENDING_REQUESTS - requestTickets.messagesPending);
-    // NOTE:  If we are already stop, the handler in outbound will fail this message for us.
-        if(enqueueMessage(inFlight, timeout, units, inFlight.getMessage().getVoltronType() != VoltronEntityMessage.Type.INVOKE_ACTION)) {
-          inFlight.sent();
-          if (inFlight.send()) {
-//  when encountering a send for anything other than an invoke, wait here before sending anything else
-//  this is a bit paranoid but it is to prevent too many resends of lifecycle operations.  Just
-//  make sure those complete before sending any new invokes or lifecycle messages
-            if (inFlight.getMessage().getVoltronType() != VoltronEntityMessage.Type.INVOKE_ACTION) {
-              inFlight.waitForAcks();
-            }
-          } else {
-            logger.debug("message not sent.  Make sure resend happens " + inFlight);
+    try {
+      msgCount.increment();
+      inflights.add(ClientConfigurationContext.MAX_PENDING_REQUESTS - requestTickets.messagesPending);
+      // NOTE:  If we are already stop, the handler in outbound will fail this message for us.
+      if (enqueueMessage(inFlight, timeout, units, inFlight.getMessage().getVoltronType() != VoltronEntityMessage.Type.INVOKE_ACTION)) {
+        inFlight.sent();
+        if (inFlight.send()) {
+          //  when encountering a send for anything other than an invoke, wait here before sending anything else
+          //  this is a bit paranoid but it is to prevent too many resends of lifecycle operations.  Just
+          //  make sure those complete before sending any new invokes or lifecycle messages
+          if (inFlight.getMessage().getVoltronType() != VoltronEntityMessage.Type.INVOKE_ACTION) {
+            inFlight.waitForAcks();
           }
         } else {
-          throwClosedExceptionOnMessage(inFlight, "Connection closed before sending message");
+          logger.debug("message not sent.  Make sure resend happens " + inFlight);
```

Comment:
logic change?  Is this another bug?

Reply:
This is me screwing up.

Reply:
Actually no it's not... it's the git diffing engine screwing things up.